content/admin/configuration/listen.md

```diff
@@ -107,9 +107,9 @@ Options:
 ## ejabberd_service
 
 Interacts with an
-	[`external component`](http://www.ejabberd.im/tutorials-transports)
+	[`external component`](https://ejabberd.im/tutorials-transports)
```

Comment:
No, because https://ejabberd.im/tutorials-transports/
is redirected to https://www.ejabberd.im/tutorials-transports/